ReactiveCocoa/NSObject+BindingTarget.swift

```diff
@@ -13,7 +13,7 @@ extension Reactive where Base: NSObject {
 	///
 	/// - returns:
 	///   A binding target that holds no strong references to the object.
-	internal func makeBindingTarget<U>(_ key: StaticString = #function, action: @escaping (Base, U) -> Void) -> BindingTarget<U> {
+	public func makeBindingTarget<U>(_ action: @escaping (Base, U) -> Void) -> BindingTarget<U> {
```

Comment:
What do you think about making the scheduler a parameter with a default value of `UIScheduler`, instead of hard-coding it?